mudslide/turbomole_model.py

```diff
@@ -233,15 +238,38 @@ def compute(self, X, couplings, gradients, reference):
         can get properly passed to Turbomole. (__init__() can get these
         file locations.)
         """
-        self.call_turbomole()
+        self.call_turbomole(outname = Path(self.workdir)/"turbo.out")
+
         self.hamiltonian = np.zeros([self.nstates(), self.nstates()])
         for i, e in enumerate(self.states):
             self.hamiltonian[i][i] = self.energies[e]
         return self.hamiltonian
 
-    def update(self, X: ArrayLike, couplings: Any = None, gradients: Any = None):
+    def update(self, X: ArrayLike, electronics: Any=None, couplings: Any = None, gradients: Any = None):
         out = cp.copy(self)
         out.position = X
+        #if electronics:
+        #    self.reference = electronics.reference
         out.update_coords(X)
         out.compute(X, couplings=couplings, gradients=gradients, reference=self.reference)
         return out
+
+    def cpc(self,dest): 
+        subprocess.run(["cpc", dest], cwd=self.workdir)
+        tmmodules_files=["control" ] 
+        for f in tmmodules_files:
+            if os.path.exists(os.path.join(os.path.abspath(self.workdir), f)) and not os.path.exists(os.path.join(dest,f)):
+                shutil.copy(os.path.join(os.path.abspath(self.workdir), f), dest)
+        
+        subprocess.run(["cpc", dest], cwd=self.workdir)
+        file_list = ['ciss_a','exspectrum', 'statistics', 'dipl_a', 'excitationlog.1', 'moments', 'vecsao', 'control', 'gradient', 'energy', 'moments' ]
```

Comment:
This looks really strange and probably doesn't work as expected. `cpc` is called twice? And the only file you check to see if you need to move is  `control`? The `file_list` at the end isn't used anywhere. If this is the code actually being run then I'm guessing your tests aren't sensitive enough to notice this failure.

Reply:
This makes more sense - I think.

<img width="1371" alt="Screen Shot 2022-11-04 at 1 52 48 PM" src="https://user-images.githubusercontent.com/46786940/200043094-0c6bc4b2-0e10-46f5-b90b-5147a69b85b7.png">